xontrib/xlsd.xsh

```diff
@@ -320,7 +321,7 @@ def _format_direntry_name(entry: os.DirEntry, show_target: bool = True) -> str:
     # Show the icon
     icon = _icon_for_direntry(entry)
     colors = []
-    name = "{}{}{{RESET}}".format(icon, name)
+    name = $XLSD_NAME_FORMAT.format(icon=icon, name=name) + "{{RESET}}"
```

Comment:
Since the `{RESET}` is no longer in a format, the double brackets aren't needed anymore
```suggestion
    name = $XLSD_NAME_FORMAT.format(icon=icon, name=name) + "{RESET}"
```